node/node_test.go

```diff
@@ -513,16 +514,31 @@ func TestMultipleGet(t *testing.T) {
 	})
 	<-added1
 
+	// commit data1
```

Comment:
Sorry I probably should have explained in a comment, the goal of this test is to commit 2 different files into a single transaction and then retrieve (Get) the files individually with 2 separate operations. So both Get operations are on the same transaction (ref) and based on the same root CID but retrieve 2 different files.